src/tactic/core.lean

```diff
@@ -2089,6 +2091,17 @@ add_tactic_doc
   decl_names               := [`tactic.mk_simp_attribute_cmd],
   tags                     := ["simplification"] }
 
+/-- A tactic to set either a basic attribute or a user attribute, as long as the user attribute has
+  no parameter.
+  Warning: if a user attribute with a parameter (that is not `unit`) is set, this will *not*
```

Comment:
Ah, yeah, I was trying to find this line. See my comment below. This warning will be ignored and people will get confused; can we enforce it, perhaps by testing immediately if `get_param` succeeds?

Reply:
Good idea! I will do that